src/org/zaproxy/zap/extension/bruteforce/BruteForceParam.java

```diff
@@ -37,19 +43,27 @@
 	private static final String RECURSIVE = "bruteforce.recursive";
 	private static final String BROWSE_FILES = "bruteforce.browsefiles";
 	private static final String FILE_EXTENSIONS = "bruteforce.fileextensions";
+	private static final String EXTENSIONS_TO_MISS = "bruteforce.extensionsToMiss";
+	private static final String FAIL_CASE_STRING = "bruteforce.failCaseString";
 	
 	public static final int DEFAULT_THREAD_PER_SCAN = 10;
 	public static final int MAXIMUM_THREADS_PER_SCAN = 200;
 	public static final boolean DEFAULT_RECURSIVE = true;
 	public static final boolean DEFAULT_BROWSE_FILES = false;
 	public static final String EMPTY_STRING = "";
+	public static final String DEFAULT_EXTENSIONS_TO_MISS = "jpg, gif, jpeg, ico, tiff, png, bmp";
+	public static final String DEFAULT_FAIL_CASE_STRING = Config.failCaseString;
+	public static final String WHITESPACE = "\\s";
+	public static final String COMMA = ",";

```

Comment:
Same here, also do we really need a constant for this? (If so, it should probably have a more meaningful name, like `EXTENSION_SEPARATOR`.)